src/content/docs/apm/apm-ui-pages/monitoring/find-performance-problems-apm-transactions-page.mdx

```diff
@@ -0,0 +1,374 @@
+
+    <table>
+      <thead>
+        <tr>
+          <th style={{ width: "200px" }}>
+            **Sort option**
+          </th>
+
+          <th>
+            **Definition**
+          </th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr>
+          <td>
+            Most time consuming (default)
+          </td>
+
+          <td>
+            The transactions that consume the most total time. This is calculated by multiplying the number of times the transaction is called by the time the transaction took to complete. In other words, a short transaction called a large number of times or a long transaction called just a few times could both appear at the top of this ranking.
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            Slowest average response time
+          </td>
+
+          <td>
+            The slowest transactions to complete. A transaction could be called only a few times, but if it is one of the longest transactions to finish, it would appear at the top of this ranking.
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            Apdex most dissatisfying
+          </td>
+
+          <td>
+            The transactions with the highest percentages of user dissatisfaction levels for the response time of your web apps and services ([Apdex](/docs/apm/new-relic-apm/apdex/apdex-measuring-user-satisfaction)).
+
+            The [dissatisfaction level](/docs/apm/new-relic-apm/apdex/view-your-apdex-score#apdex-dissat) is the difference between a perfect Apdex score (1.0) and your app's `apdex_t` ("tolerating") score, based on your [Apdex settings](/docs/apm/new-relic-apm/apdex/change-your-apdex-settings). Transactions that have the highest (worst) percentages appear at the top of this ranking.
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            Throughput (requests per minute)
+          </td>
+
+          <td>
+            The amount of [throughput](/docs/using-new-relic/welcome-new-relic/getting-started/glossary#throughput) in operation requests per minute (RPM). Transactions that have the highest RPM appear at the top of this ranking.
+          </td>
+        </tr>
+
+      </tbody>
+    </table>
+  </Collapser>
+
+  <Collapser
+    id="chart-views"
+    title="View transaction data in different visual formats"
+  > 
+    ![Screenshot showing selectors to change the chart.](./images/chart_type_selectors.png "chart_type_selectors.png")
+
+    The **Transaction** page's **Top web transactions** and **Top non-web transactions** charts, as well the **App performance** window for a selected transaction include options to view data as a [chart](/docs/using-new-relic/user-interface-functions/view-your-data/select-chart-views), [histogram](/docs/using-new-relic/user-interface-functions/view-your-data/histograms-view-data-distribution), or [percentile](/docs/using-new-relic/user-interface-functions/view-your-data/percentiles-compare-ranked-data). 
```

Comment:
```suggestion
    The **Transaction** page's **Top web transactions** and **Top non-web transactions** charts, as well the **App performance** window for a selected transaction include options to view an [average response time breakdown](/docs/using-new-relic/user-interface-functions/view-your-data/select-chart-views), [histogram](/docs/using-new-relic/user-interface-functions/view-your-data/histograms-view-data-distribution), or [response time percentiles](/docs/using-new-relic/user-interface-functions/view-your-data/percentiles-compare-ranked-data). 
```

Reply:
I agree with this feedback but am having a little trouble figuring out how to translate into the needed edit. Will reach out so we can schedule some time, @rkbodenner.